tests/Core/ArcTable.Tests.fs

```diff
@@ -1863,6 +1863,40 @@ let private tests_RemoveColumns =
             Expect.equal table.Values.[(table.ColumnCount-1,table.RowCount-1)] (CompositeCell.createTerm oa_SCIEXInstrumentModel) "table.ColumnCount-1,table.RowCount-1"
         )
     ]
+
+let private tests_TryGetColumnByHeaderBy =
+    testList "TryGetColumnByHeaderBy" [                 
+        testCase "on empty column" (fun () ->
```

Comment:
The two test case names are switched up. Also rename to "Column with values" and "Empty column"